ShuttleTrackeriOS/View/CustomTabBar.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CustomTabBar.swift
+//  ShuttleTrackeriOS
+//
+//  Created by Beiqi Zou on 3/18/19.
+//  Copyright © 2019 WTG. All rights reserved.
+//
+
+import UIKit
+
+class CustomTabBar: UITabBarController {
```

Comment:
Does this get used anywhere else? is it possible to just make another existing class extend UITabBarController instead of creating a new class?

Reply:
I don't think I could extend UITabBarController because I changed the custom class of tab bar controller in the storyboard
<img width="312" alt="Screen Shot 2019-03-19 at 12 07 46 PM" src="https://user-images.githubusercontent.com/34224013/54622239-9da8ef00-4a3f-11e9-8a8f-1b6387a610c0.png">